index.html

```diff
@@ -469,13 +493,220 @@ <h4>Attributes</h4>
               </p>
             </li>
             <li>
-              <p>Return
-              <var>track</var>.{{MediaStreamTrack/[[VideoStats]]}}.</p>
+              <p>Return <var>track</var>.{{MediaStreamTrack/[[Stats]]}}.</p>
             </li>
           </ol>
         </dd>
       </dl>
     </section>
+  </section>
+    <section>
+      <h4>The MediaStreamTrackAudioStats interface</h4>
+      <div>
+        <pre class="idl" data-cite="HR-TIME">[Exposed=Window]
+interface MediaStreamTrackAudioStats {
+  readonly attribute unsigned long long deliveredFrames;
+  readonly attribute DOMHighResTimeStamp deliveredFramesDuration;
+  readonly attribute DOMHighResTimeStamp deliveredFramesLatency;
+  readonly attribute DOMHighResTimeStamp? latency;
+  readonly attribute unsigned long long totalFrames;
+  readonly attribute DOMHighResTimeStamp totalFramesDuration;
+  [Default] object toJSON();
+};
+        </pre>
+        <div>
+          <p>The {{MediaStreamTrackAudioStats}} expose frame counters for the
+          {{MediaStreamTrack}} that created it. For this track, the user agent
+          is required to count each audio frame from its source as follows:</p>
+          <ul>
+            <li>
+              <p>A frame is considered a <dfn data-lt="delivered audio frames">
+              delivered audio frame</dfn> if it either was delivered to a sink
+              or would have been delivered to a sink, if one was connected.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames duration">delivered
+              audio frames duration</dfn> is the total duration of all
+              [= delivered audio frames =]. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds.</p>
+            </li>
+            <li>
+              <p>The <dfn data-lt="delivered audio frames latency">delivered
+              audio frames latency</dfn> is the sum of [= delivered audio
+              frames =]'s input latency. The input latency is the estimated
+              latency of the capture device plus the time it takes the user
+              agent to deliver that frame. This measurement is incremented at
+              the same time as [= delivered audio frames =] and is measured in
+              milliseconds. Note that once a frame has been delivered to a sink,
+              that sink may internally have its own latency (such as playout
+              delay) which is not included in this measurement.</p>
```

Comment:
Summing latencies of audio buffer (if that's what you mean) does not make any sense. You want a series of values here if you want to understand what's going on.

Reply:
> You want a series of values here if you want to understand what's going on.

Is that perhaps covered by #119?

@henbos would one way to progress here be to split out the latency metric in a separate PR (maybe for #119) so this PR with the audio stats model can be merged?

Reply:
For the sake of progress I'll remove all latency metrics and follow up on them in #119.

It seems to me that the "controversial" part is the latency so let's merge the frame counters and follow up. The second part is about how to phrase the algorithm ("stable state" or otherwise) but that's #108 and editorial so let's do that for both audio and video when we have decided what phrase to use.

The remaining counters are very close to what we already have for video.